charts/linkerd2/templates/web-rbac.yaml

```diff
@@ -103,6 +103,36 @@ subjects:
   name: linkerd-web
   namespace: {{.Namespace}}
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: linkerd-web-check-injector
+  namespace: default
+  labels:
+    {{.ControllerComponentLabel}}: web
+    {{.ControllerNamespaceLabel}}: {{.Namespace}}
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["create"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: linkerd-web-check-injector
+  namespace: default
```

Comment:
I wonder whether it makes sense for this to be configurable to allow users to specify a different "test" namespace

Reply:
Good point @zaharidichev, that could be interesting.

Reply:
I would prefer to keep `check` as zero config right now. I think that if someone does a skip they know what they're doing (and we can address that through the docs).

Reply:
As an added note, this check is primarily for users who have api-servers that cannot call into the cluster because of firewall rules (GKE private clusters, unique EKS deploys). A `default` check will verify that and be good enough (tm) for most folks.